### DISCUSSION TITLE
关于在夹杂的英文中使用标点符号的疑问

### DISCUSSION THREAD

**Discussion by iBug** [sparanoid/chinese-copywriting-guidelines#148] [Q&A]
[使用全角中文标点](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-CN.md#%E4%BD%BF%E7%94%A8%E5%85%A8%E8%A7%92%E4%B8%AD%E6%96%87%E6%A0%87%E7%82%B9) 一节指出了如下 “正确” 用法：

> 核磁共振成像（NMRI）是什么原理都不知道？JFGI！

但是我在实际使用中发现如下规则更好：

- 对于词末/句末的分隔类标点，与其前面的字符保持一致：使用半角字符当且仅当其前面为半角字符（半角字母数字等）。例如：

  > Good: 核磁共振成像一般简写为 NMRI.
  >
  > Bad: 核磁共振成像一般简写为 NMRI。

- 对于配对的标点（括号、引号等），当其内部的第一个和最后一个字符都是半角字符时，使用半角符号（注意与前后的文字保持空格间距）：

  > Good: 核磁共振成像 (NMRI) 是什么原理
  >
  > Bad: 核磁共振成像（NMRI）是什么原理  
  > Bad: 核磁共振成像(NMRI)是什么原理

  - 特殊情况：单个字母或数字表示选项或类似含义时，可以使用全角符号

    > Good: （A 或 B）  
    > Also good: (A 或 B)

  - 边界情况：若配对标点中内容的第一个和最后一个字符为全角半角各一个时，根据内容主体，两种括号均可：

    > Good: NMRI (Nuclear Magnetic Resonance Imaging, 核磁共振成像) 是……  
    > Also good: NMRI（Nuclear Magnetic Resonance Imaging, 核磁共振成像）是……

  - 例外：直引号「」不使用本规则

**Comment by liyishuai** [comment]
根据中华人民共和国新闻出版行业标准CY/T 154—2017号《中文出版物夹用英文的编辑规范》
第5.1.2.1节：
> 中文陈述句内夹用英文单词或词组，该句子应以中文句号结尾。
示例1：medium的复数形式不一定是media。

第5.8.2.2节：
> 中文句子内夹用英文句子时，该英文句子用中文引号标示，全句句末应使用中文标点。
示例1：见面就问“How old are you?”是不恰当的。
示例2：为什么要告诉所有的人“The door is left unlocked.”？

第5.9.2.1节：
> 中文句子内夹用了以注释、补充或说明的英文句子或语段，该英文句子或语段用中文圆括号标示。
示例2：DIY（Do It Yourself）教育有助于孩子独立人格的形成。
示例3：这次会议被定性为“高峰论坛”（summit forum）。

在普通话、简体中文语境下，反对你提出的建议。

**Comment by xslidian** [comment]
半角括号+空格属微软体系流传下来的习惯，只是为了照顾 UI 美观，通常不建议用于网页正文。

原则上标点跟随它所属的成分，因此「简写为 NMRI.」是大脑很混乱的表现。但「全场：WTF?!」完全没问题。

PS：英文标点乱放属西方体例，东方文字为主的排版不要学他们。
